### DISCUSSION TITLE
Install Spyder under Virtual Environment (Windows 10 system)

### DISCUSSION THREAD

**Discussion by meckwell** [pybamm-team/PyBaMM#2087] [Q&A]
Dear PyBaMM Team,

Recently I downloaded and installed the latest version of PyBaMM under a newly created virtual environment (Windows 10 system). After that, I tried to install Spyder under the virtual environment but it stuck at "connecting to kernel" stage (See attached).

I was wondering if anyone had this issue recently? ( Note: previously I have successfully installed spyder in my previous virtual environment before, so It should be doable.)

@TomTranter Hi Tom I remember you use the Windows system as well. Have you ever encountered a similar problem?

Many thanks for your help and time again!
![Spyder problem](https://user-images.githubusercontent.com/33289036/172209026-870044ae-35ec-49b4-a396-01b3331b931c.PNG)

**Comment by brosaplanella** [comment]
That seems to be a spyder issue. Have you tried getting in touch with the spyder community?

**Reply by TomTranter** [reply]
I use conda to manage environments and don't have any issues with spyder

**Reply by meckwell** [reply]
Sorry for not getting back to you sooner as I just returned from leave. Thank you Ferran and Tom for your advice. I will act accordingly!
